WORKSPACE

```diff
@@ -59,11 +86,13 @@ ghc_version = "8.4.4"
 # A GHC from a bindist for Windows
 ghc_bindist(
     name = "ghc_windows",
+    target = "windows_amd64",
     version = ghc_version,
 )
 
 register_toolchains(
-    "//tests:ghc",
+    "//tests:ghc_linux",
+    "//tests:ghc_osx",
```

Comment:
no `ghc_window`?

Reply:
No. See the commit messages as for why. Windows uses `ghc_bindist` and that rule self registers.